cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js

```diff
@@ -29,29 +21,59 @@ export default ({ service, variant, pageType, path }) => {
-    getCookieBanner(service, variant).should('not.be.visible');
+    getCookieBannerAmp(service, variant).should('not.be.visible');
   });
 
   it('should not show privacy & cookie banners once both accepted, on reload', () => {
     getPrivacyBannerAccept(service, variant).click();
-    getCookieBannerAccept(service, variant).click();
+    getCookieBannerAcceptAmp(service, variant).click();
 
     visitPage(path, pageType);
 
     getPrivacyBanner(service, variant).should('not.be.visible');
-    getCookieBanner(service, variant).should('not.be.visible');
+    getCookieBannerAmp(service, variant).should('not.be.visible');
   });
 
-  it('should not show privacy & cookie banners once cookie banner declined, on reload', () => {
+  it('should go to manage settings banner once manage settings button is clicked, and when accept button is clicked the banners no longer show', () => {
     getPrivacyBanner(service, variant).should('be.visible');
-    getCookieBanner(service, variant).should('not.be.visible');
+    getCookieBannerAmp(service, variant).should('not.be.visible');
 
     getPrivacyBannerAccept(service, variant).click();
-    getCookieBannerReject(service, variant).click();
+    getCookieBannerManageSettingsButton(service, variant).click();
+    getCookieBannerAcceptInManageSettings(service, variant).click();
+    getCookieBannerManageSettings(service, variant).should('not.be.visible');
 
-    visitPage(path, pageType);
+    cy.reload();
+
+    getPrivacyBanner(service, variant).should('not.be.visible');
+    getCookieBannerAmp(service, variant).should('not.be.visible');
+  });
+
+  it('should go to manage settings banner once manage settings button is clicked, and when reject button is clicked the banners no longer show', () => {
+    getPrivacyBanner(service, variant).should('be.visible');
+    getCookieBannerAmp(service, variant).should('not.be.visible');
+
+    getPrivacyBannerAccept(service, variant).click();
+    getCookieBannerManageSettingsButton(service, variant).click();
+    getCookieBannerRejectInManageSettings(service, variant).click();
+    getCookieBannerManageSettings(service, variant).should('not.be.visible');
+
+    cy.reload();
 
     getPrivacyBanner(service, variant).should('not.be.visible');
-    getCookieBanner(service, variant).should('not.be.visible');
+    getCookieBannerAmp(service, variant).should('not.be.visible');
+  });
+
+  it('should show privacy banner once you reload the page on the Manage settings expanded view without clicking Accept or Reject buttons', () => {
+    getPrivacyBanner(service, variant).should('be.visible');
+    getCookieBannerAmp(service, variant).should('not.be.visible');
+
+    getPrivacyBannerAccept(service, variant).click();
+    getCookieBannerManageSettingsButton(service, variant).click();
+    getCookieBannerManageSettings(service, variant).should('be.visible');
+
+    cy.reload();
+
+    getPrivacyBanner(service, variant).should('be.visible');
```

Comment:
Did you confirm this behaviour is correct/acceptable with @aeroplanejane, I think it differs from canonical where:
- The privacy banner shows and immediately drops a cookie
- Reload the page
- The privacy banner does not show as the cookie has been dropped
?

I think @simonsinclair suggested it may not be possible on amp but just checking it is known to product at the very least.